.ci/linux/deploy.sh

```diff
@@ -4,9 +4,17 @@
 #
 set -euo pipefail
 
-for nupkg in Elastic.Apm Elastic.Apm.AspNetCore Elastic.Apm.EntityFrameworkCore Elastic.Apm.NetCoreAll Elastic.Apm.EntityFramework6 Elastic.Apm.AspNetFullFramework
+declare -a projectsToPublish=("Elastic.Apm" "Elastic.Apm.AspNetCore" "Elastic.Apm.EntityFrameworkCore" "Elastic.Apm.NetCoreAll" "Elastic.Apm.EntityFramework6" "Elastic.Apm.AspNetFullFramework")
+
+for project in  "${projectsToPublish[@]}"
 do
-	nupkg+=".nupkg"
-	echo "dotnet nuget push ${nupkg}"
-	dotnet nuget push ${nupkg} -k ${1} -s ${2}
+	for nupkg in $(find . -type f -not -path './.nuget/*' -name '*.nupkg')
+	do
+		pattern=".*${project}[0-9|.]*.nupkg"
```

Comment:
`[0-9|.]*` could be improved since for let's say `Elastic.Apm` i'd also match `Elastic.Apm123`, but that's fine here.